internal/configs/version2/nginx.virtualserver.tmpl

```diff
@@ -1,6 +1,6 @@
 {{ range $u := .Upstreams }}
 upstream {{ $u.Name }} {
-    zone {{ $u.Name }} 256k;
+    {{if ne $u.UpstreamZoneSize "0"}}zone {{$u.Name}} {{$u.UpstreamZoneSize}};{{end}}
```

Comment:
consistency problem mentioned in the comment for `nginx-plus.virtualserver.tmpl`